core/src/components/input/input.ionic.outline.scss

```
@@ -0,0 +1,110 @@
+@import "./input.vars";
+
+// Input Fill: Outline (Ionic Theme)
+// ----------------------------------------------------------------
+
+:host(.input-fill-outline) {
+  --border-radius: 4px;
+  --padding-start: 12px;
+  --padding-end: 12px;
+  --placeholder-color: #535353;
+  --placeholder-opacity: 1;
+}
+
+/**
+ * The bottom content should never have
+ * a border with the outline style.
+ */
+:host(.input-fill-outline) .input-bottom {
+  border-top: none;
+}
+
+:host(.input-fill-outline) .input-wrapper {
+  /**
+   * For the ionic theme, the padding needs to sit on the
+   * native wrapper instead, so that it sits within the
+   * outline container but does not affect the label text.
+
+   * For the ionic theme, the horizontal padding needs to
+   * sit on the native wrapper instead, so that
+   */
+  @include padding(0);
+
+  /**
+   * Outline inputs do not have a bottom border.
+   * Instead, they have a border that wraps the
+   * input + label.
+   */
+  border-bottom: none;
+}
+
+:host(.input-fill-outline.input-label-placement-stacked) .label-text-wrapper {
+  @include transform-origin(start, top);
+
+  /**
+   * Label text should not extend
+   * beyond the bounds of the input.
+   */
+  max-width: calc(100% - var(--padding-start) - var(--padding-end));
+}
+
+:host(.input-fill-outline) .label-text-wrapper {
+  /**
+   * The label should appear on top of an outline
+   * container that overlaps it so it is always clickable.
+   */
+  position: relative;
+
+  color: #373737;
```

This hex is an Ionic design token:`ionic-color-neutral-700`. I'm not sure if the branch has the latest from https://github.com/ionic-team/ionic-framework/pull/29245. If it does then we should be able to do:
```suggestion
  --color: $ionic-color-neutral-700;
```

Also it should be using the `--color` to override.

---

Token added in https://github.com/ionic-team/ionic-framework/pull/29268/commits/8d3bc577de78188b0839b850a6891e40e9a16929. The `--color` variable is only [used on the host](https://github.com/ionic-team/ionic-framework/blob/54247d1e8c19f8af0663873dc7eebfa6295b83dd/core/src/components/input/input.scss#L66), meaning that (1) setting `--color` on the label element doesn't do anything, and (2) setting it on the host instead would also affect the color of the input value.